cypress/e2e/tests/migration/applicationinventory/assessment/miscellaneous.test.ts

```diff
@@ -231,6 +231,30 @@ describe(["@tier3"], "Tests related to application assessment and review", () =>
         archetypes[0].delete();
         deleteByList(tags);
     });
+    it("Deletes assessments from an archived questionnaire from an archetype and application", function () {
```

Comment:
Maybe "Delete Assessments of Archived Questionnaire from it's Associated Archetype and Application"

Reply:
do you think changing it to  it("Deletes assessments from archived questionnaire associated with an archetype and application" is good?

Reply:
yea I think it's more descriptive.

Reply:
done